src/Exception/AccountNotValidatedException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace AppBundle\Exception;
+
+use AppBundle\Entity\Adherent;
+use Symfony\Component\Security\Core\Exception\AccountStatusException;
+use Throwable;
+
+class AccountNotValidatedException extends AccountStatusException
+{
+    public function __construct(Adherent $adherent, int $code = 0, Throwable $previous = null)
+    {
+        parent::__construct(sprintf('Account[%s] tried to connect without validated his email.', $adherent->getEmailAddress()), $code, $previous);
```

Comment:
I think this message is useless, the `ExceptionListener` will log the exception entirely.